### ISSUE TITLE
Footer: Edit My Links not working in Internet Explorer

### ISSUE THREAD

**Issue by BradNailer** [SharePoint/sp-starter-kit#102]
#### Category
- [X] Question
- [X] Bug
- [ ] Enhancement

#### Expected or Desired Behavior

The Edit my links feature to open when clicked while using Internet Explorer 11.  Is Internet Explorer supported with the SP Starter Kit?

#### Observed Behavior

When the Edit link is clicked a small white rectangle is shown, the screen goes grey for a second, and the Edit my Links window does not appear. 

Also, there is a graphical fault in the the footer area that only appears with using IE. Shown below:
![spstarterkit_footer_bug](https://user-images.githubusercontent.com/41487313/43087315-63b0137a-8e54-11e8-9804-b446c3b3e206.JPG)

#### Steps to Reproduce

Try to click Edit in the footer when using Internet Explorer.

Thanks for you time.

**Comment by PaulMac66** [comment]
Hi Brad.

Did you ever resolve this issue? I am having the same trouble.

**Comment by BradNailer** [comment]
Sorry Paul

I have not been able to solve this as of yet.

Thanks

________________________________

**Comment by mschaefer85** [comment]
Experiencing the same issue, unfortunately. The dialog opens, but closed immediately.

**Comment by nhadro** [comment]
Has anyone been able to resolve this issue?

**Comment by trevorhuinker** [comment]
> Has anyone been able to resolve this issue?

Same here... does anyone have an update? @VesaJuvonen

**Comment by Laskewitz** [comment]
Not working for me either. Would be great to have a solution for this.

**Comment by simonagren** [comment]
These are the steps I had to make with the newest version of the starterkit. I needed the PNPJs Polyfills for the footer to show in IE11, you might not have to do that.
And then what was breaking when pressing "Edit" was the usage of `new Event`, which doesn't work in IE.
So I changed that code.


1. Open Node, cd into `\sp-starter-kit\solution`
 - Run:
```
npm install --save @pnp/polyfill-ie11
```

2. Open solution in VS Code. And go into the portal footer.
solution\src\extensions\portalFooter\PortalFooterApplicationCustomizer.ts
Then add:
import "@pnp/polyfill-ie11"; 

3. Go to this `solution\src\common\myLinks\MyLinksDialog.tsx`
And on line 108 change:  window.dispatchEvent(new Event('resize'));
To: 
```TypeScript
let resizeEvent = window.document.createEvent('UIEvents'); 
resizeEvent.initUIEvent('resize', true, false, window, 0); 
window.dispatchEvent(resizeEvent); 
```

4. Save and then run in Node: 
gulp bundle --ship && gulp package-solution --ship 

5. Then go to you app catalog and upload and deploy the new package:
sp-starter-kit\solution\sharepoint\solution
